img/v/veeam.com.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 228 228"><path fill="#01B137" d="M149 229H1V1h228v228"/><path fill="#F9FDFB" d="M145.4 163c-6.4 12.3-8.9 15.2-11.3 18-19.2 11-33.8 6.2-42-4.5-49.8-98.7-52.7-103.7-55-109-3.2-7.5 1-17 8.3-20.4 7.8-3.5 16.6-.8 20.4 7 35.1 74 39 83 43.4 91.7 1.2 2.2 3.3 4.8 5.5 5.5 3.9 1.2 4.8-2.8 6-5.3 39.8-84.4 41.3-88 43-91.3 4.5-8.2 13.6-11.4 21.1-7.5 8 4.2 11.1 13.9 7 22.2z"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 102.18 39.79"><defs><linearGradient id="a" x1=".32" x2="99.69" y1="21.74" y2="17.35" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#007f49"/><stop offset=".34" stop-color="#00d15f"/><stop offset=".64" stop-color="#32f26f"/><stop offset="1" stop-color="#9cffa3"/></linearGradient></defs><path fill="url(#a)" d="M19.53 33.99 0 14.57 14.57 0l33.78 33.67a3.95 3.95 0 0 0 5.49 0L87.61 0l14.57 14.57L82.65 34.1a19.5 19.5 0 0 1-13.72 5.7H33.25a19.8 19.8 0 0 1-13.72-5.81"/></svg>
```
Found this logo/icon in their [branding assets](https://www.veeam.com/company/brand-resource-center.html), looks like they updated their logo recently.

Reply:
Thanks for the correction, I did find its other icons online, because I think this one looks simpler! @fletcherdodd